package.json

```diff
@@ -4,6 +4,8 @@
   "private": true,
   "dependencies": {
     "@emailjs/browser": "^3.11.0",
+    "@emotion/react": "^11.11.3",
+    "@emotion/styled": "^11.11.0",
```

Comment:
Required for MUI

Reply:
Checkout #13 for the styled component version of the MUI implementation